utils/check-missing-logos.js

```diff
@@ -0,0 +1,46 @@
+const {
+  readQuickstartFile,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+/**
+ * Validate that a quickstart has a logo image
+ * @param {String[]} quickstartDirs - The directories of the quickstarts
+ * @returns {String[]} - The directories of the quickstarts that do not have a logo image
+ */
+const validateLogoExists = (quickstartDirs) => {
+  const noLogos = quickstartDirs
+    .map((quickstart) => {
+      const quickstartConfig = readQuickstartFile(quickstart).contents[0];
+      
+      if (!quickstartConfig.logo) {
+        return quickstart.match(/newrelic-quickstarts(.*)/)[1];
+      }
+      
+      
+    }).filter(Boolean);
+  return noLogos;
+};
+
+
+const main = () => {
+  const quickstartDirs = findMainQuickstartConfigFiles();
+  const quickstartsWithNoLogos = validateLogoExists(quickstartDirs)
+  if (quickstartsWithNoLogos.length > 0) {
```

Comment:
I think the "quickstarts" qualifier for these variables might be a little too much context. I think we can assume that they're all related to quickstarts in this script.

Reply:
that is a good point ye